.drone.star

```diff
@@ -1035,19 +1098,20 @@ def buildOcisPhoenix():
 		}],
 	}]
 
-def ocisPhoenixService():
+def ocisPhoenixService(glauth = False):
 	return[{
 		'name': 'phoenix',
 		'image': 'webhippie/golang:1.13',
 		'pull': 'always',
 		'detach': True,
 		'environment' : {
-			'PHOENIX_WEB_CONFIG': '/srv/config/drone/ocis-config.json',
-			'PHOENIX_ASSET_PATH': '/var/www/owncloud/phoenix/dist'
+			'PHOENIX_WEB_CONFIG': '/srv/config/drone/config.json' if glauth else '/srv/config/drone/ocis-config.json',
+			'PHOENIX_ASSET_PATH': '/var/www/owncloud/phoenix/dist',
+			'PHOENIX_OIDC_CLIENT_ID': 'phoenix'
 		},
 		'commands': [
 			'cd /var/www/owncloud',
-			'./ocis-phoenix server',
+			'./ocis-phoenix --log-level debug server',
```

Comment:
do we want to keep that high log-level?

Reply:
Yes, I think we should run all the ocis services in debug log level